pkg/sources/github/github.go

```diff
@@ -951,11 +964,8 @@ func (s *Source) addAllVisibleOrgs(ctx context.Context) {
 		},
 	}
 	for {
-		orgs, resp, err := s.apiClient.Organizations.ListAll(ctx, orgOpts)
-		if err == nil {
-			resp.Body.Close()
-		}
```

Comment:
Closing the response body isn't necessary because `go-github` does it for us.

https://github.com/google/go-github/blob/c135be9459ed40979be841f065badb7de19d467b/github/github.go#L902